custom/icds_reports/queries.py

```diff
@@ -26,9 +26,10 @@ def get_test_district_locations_id(domain):
 
 
 def get_cas_data_blob_file(indicator, location, date):
-    indicators = ['', 'child_health_monthly', 'ccs_record_monthly', 'agg_awc']
+    if indicator not in ['', 'child_health_monthly', 'ccs_record_monthly', 'agg_awc']:
+        raise ValueError('Bad indicator {}'.format(indicator))
```

Comment:
Should it be good to catch the exception in the caller function.(It might not be needed as this url is trigger using UI and has less chance raising this error.

Reply:
I kind of view this as an assertion which was why I didn't bother. E.g. you can already trigger a similar error today by accessing [a number greater than the list index](https://www.icds-cas.gov.in/a/icds-cas/cas_export?indicator=5&location=039bbe4a40de499ea87b9761537dd611&month=9&year=2019), (or [not a number at all](https://www.icds-cas.gov.in/a/icds-cas/cas_export?indicator=foobar&location=039bbe4a40de499ea87b9761537dd611&month=9&year=2019))

Unless we strive to not have any of these URLs 500? (In which case that's probably a larger exercise...)